composer.json

```diff
@@ -17,16 +17,20 @@
         "doctrine/common": "^2.7.1"
     },
     "require-dev": {
+        "doctrine/coding-standard": "^1.0",
         "phpunit/phpunit": "^6.3",
         "phpunit/phpunit-mock-objects": "!=3.2.4,!=3.2.5",
+        "squizlabs/php_codesniffer": "^3.0",
         "symfony/console": "2.*||^3.0",
-        "doctrine/coding-standard": "^1.0",
-        "squizlabs/php_codesniffer": "^3.0"
+        "symfony/phpunit-bridge": "^3.3"
```

Comment:
Adding this so that we get a report of deprecations at the end of the test suite.

Reply:
Is everyone on board with this?

Reply:
What's the purpose of reporting the usage of deprecated code in a test run? As far as I understand, the code is deprecated for the outside usage but the test still has to be tested until removed entirely.

Reply:
@morozov you are right in that in needs to be tested. Such tests are supposed to be marked with a dedicated annotation `@group legacy`, so that it does not make the build fail. But if somewhere in the lib, we call the deprecated API ourselves, we must be aware of it, and change that. If we don't users, will get a deprecation *and won't be able to do anything about it*. That's what this is about.

Reply:
@greg0ire thanks for the explanation. Makes sense.